README.md

```diff
@@ -1,62 +1,24 @@
-# Conduit Connector Template
-This is a template project for building [Conduit](https://conduit.io) connectors in Go. It makes it possible to
-start working on a Conduit connector in a matter of seconds.
+# Conduit Connector for gRPC Client
+The gRPC Client connector is one of [Conduit](https://conduit.io) plugins. It provides a destination gRPC Client connector.
```

Comment:
I'd suggest a few words about that it's supposed to be connecting to another Conduit instance, at least for now.

Reply:
I added something for now, but was thinking about adding a diagram that shows how the whole flow should look, will keep that in mind for later.